src/shared/components/palette/base-field/field-write.component.ts

```diff
@@ -58,5 +58,9 @@ export class FieldWriteComponent extends AbstractFieldWriteComponent implements
     }
     component.instance['isExpanded'] = this.isExpanded;
     this.fieldContainer.insert(component.hostView);
+
+    // EUI-3267.
+    // Set up the flag for whether this can have a grey bar.
+    this.canHaveGreyBar = this.caseField.show_condition && this.caseField.field_type.type !== 'Collection';
```

Comment:
Do we need to check for truthy of caseField? Or its always populated?

Reply:
Theoretically, but if `caseField` is null, it will have gone wrong before it gets to this bit of code as prior lines use properties of it without a null check. There's also code above to instantiate a `CaseField` if it's not of the correct type.